configs/virtual.js

```diff
@@ -34,12 +34,15 @@ exports.options = [
     // These are the PluginOptions passed into the plugin's
     // constructor.
     'pluginOptions': {
+      'connector': 'http://localhost:4000',
       'prefix': 'test.nerd.',
       'account': 'nerd',
       'host': 'ws://broker.hivemq.com:8000',
-      'limit': '0',
-      'max': '1000',
-      'balance': '100',
+      'minBalance': '0',
+      'maxBalance': '1000',
+      'initialBalance': '100',
+      'settleIfUnder': '0',
+      'settleIfOver': '1000',
```

Comment:
As discussed, this kind of setup stuff should definitely move into the plugin repo itself